### PR TITLE
Tweak the embed page CSS so the top content sits better

### LINKED ISSUE #4460: Terms screen elements overlap
**Describe the bug**
Sorry if it was already reported, haven't found it.

Screenshot 2020-05-28 at 13 36 52

**To Reproduce**

1. Go to Products > Attributes > Configure terms
2. See overlap

**System status report**
<details>
`
### WordPress Environment ###

WordPress address (URL): http://one.wordpress.test
Site address (URL): http://one.wordpress.test
WC Version: 4.3.0
REST API Version: ✔ 1.0.8
WC Blocks Version: ✔ 2.5.16
Action Scheduler Version: ✔ 3.1.6
WC Admin Version: ✔ 1.2.3
Log Directory Writable: ✔
WP Version: 5.4.1
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: nginx/1.17.10
PHP Version: 7.4.6
PHP Post Max Size: 1 GB
PHP Time Limit: 30
PHP Max Input Vars: 1000
cURL Version: 7.58.0
OpenSSL/1.1.1d

SUHOSIN Installed: –
MySQL Version: 5.5.5-10.3.23-MariaDB-1:10.3.23+maria~bionic
Max Upload Size: 1 GB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 4.3.0
WC Database Prefix: wp_
Total Database Size: 23.27MB
Database Data Size: 12.99MB
Database Index Size: 10.28MB
wp_woocommerce_sessions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_items: Data: 0.28MB + Index: 0.13MB + Engine InnoDB
wp_woocommerce_order_itemmeta: Data: 2.52MB + Index: 3.03MB + Engine InnoDB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wctracker_snapshots: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_actionscheduler_actions: Data: 1.02MB + Index: 0.36MB + Engine InnoDB
wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_logs: Data: 1.02MB + Index: 0.19MB + Engine InnoDB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_comments: Data: 0.16MB + Index: 0.16MB + Engine InnoDB
wp_gf_draft_submissions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_gf_entry: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_gf_entry_meta: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_gf_entry_notes: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_gf_form: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_gf_form_meta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_gf_form_revisions: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_gf_form_view: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailchimp_carts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mailchimp_jobs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_options: Data: 2.06MB + Index: 0.08MB + Engine InnoDB
wp_postmeta: Data: 2.52MB + Index: 4.03MB + Engine InnoDB
wp_posts: Data: 1.52MB + Index: 0.22MB + Engine InnoDB
wp_termmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_terms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_term_relationships: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_usermeta: Data: 0.39MB + Index: 0.44MB + Engine InnoDB
wp_users: Data: 0.05MB + Index: 0.05MB + Engine InnoDB
wp_wc_admin_notes: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 0.06MB + Index: 0.03MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 0.42MB + Index: 0.53MB + Engine InnoDB
wp_wc_order_stats: Data: 0.09MB + Index: 0.05MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 0.08MB + Index: 0.06MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB

### Post Type Counts ###

attachment: 91
page: 13
post: 5
product: 93
product_variation: 45
revision: 33
shop_coupon: 6
shop_order: 638
shop_order_refund: 6

### Security ###

Secure connection (HTTPS): ❌
					Your store is not using HTTPS. Learn more about HTTPS and SSL Certificates.
Hide errors from visitors: ✔

### Active Plugins (8) ###

Query Monitor: by John Blackbourn – 3.5.2
Mailchimp for WooCommerce: by Mailchimp – 2.3.4 – Installed version not tested with active version of WooCommerce 4.3.0
RTL Tester: by <a href="http://blog.yoavfarhi.com">Yoav Farhi</a>
<a href="http://automattic.com">Automattic</a> – 1.2

WooCommerce Smooth Generator: by Automattic – 1.0.0 – Installed version not tested with active version of WooCommerce 4.3.0
WooCommerce Services: by Automattic – 1.23.0 – Installed version not tested with active version of WooCommerce 4.3.0
WooCommerce: by Automattic – 4.3.0-dev
WP Crontrol: by John Blackbourn & crontributors – 1.8.3

### Inactive Plugins (9) ###

Jetpack by WordPress.com: by Automattic – 8.1.1
Variation Swatches for WooCommerce: by Emran Ahmed – 1.0.81 – Installed version not tested with active version of WooCommerce 4.3.0
W3 Total Cache: by BoldGrid – 0.11.0
WooCommerce - ShipStation Integration: by WooCommerce – 4.1.30 (update to version 4.1.36 is available) – Installed version not tested with active version of WooCommerce 4.3.0
WooCommerce Admin: by WooCommerce – 1.1.1 – Installed version not tested with active version of WooCommerce 4.3.0
WooCommerce PayPal Checkout Gateway: by WooCommerce – 1.6.17 (update to version 2.0.2 is available) – Installed version not tested with active version of WooCommerce 4.3.0
WooCommerce PayPal Powered by Braintree Gateway: by WooCommerce – 2.3.3 (update to version 2.3.10 is available) – Installed version not tested with active version of WooCommerce 4.3.0
WooCommerce Stripe Gateway: by WooCommerce – 4.3.1 – Installed version not tested with active version of WooCommerce 4.3.0
WooCommerce Subscriptions: by Automattic – 2.6.5 – Installed version not tested with active version of WooCommerce 4.3.0

### Dropin Plugins (1) ###

db.php: Query Monitor Database Class

### Settings ###

API Enabled: –
Force SSL: –
Currency: GBP (£)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
subscription (subscription)
variable (variable)
variable subscription (variable-subscription)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –

### WC Pages ###

Shop base: #6 - /shop/
Cart: #7 - /cart/
Checkout: #8 - /checkout/
My account: #9 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: Storefront
Version: 2.5.3 (update to version 2.5.6 is available)
Author URL: https://woocommerce.com/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: storefront/woocommerce/emails/admin-new-order.php

### Action Scheduler ###

Complete: 962
Oldest: 2020-04-28 14:07:07 +0000
Newest: 2020-05-27 13:04:46 +0000

Pending: 2
Oldest: 2020-05-27 13:30:40 +0000
Newest: 2020-05-27 14:47:55 +0000

`
</details>

### PR DESCRIPTION
**Pull request by becdetat**
Fixes #4460

The ticket mentions the  Help and Screen Options buttons overlapping the terms screen elements. The problem was actually deeper than that - the entire page header was being obscured by the WooCommerce layout header (the white bar).

This tweaks the embed page styles so the elements fit correctly. A nice side effect of these changes for the terms screen was that many of the other WC screens sit more nicely as well - previously there was a large gap between the WooCommerce layout header and the page content.

#### Product tags
Before:
image

After:
image

#### Reports
Before:
image

After:
image

### Detailed test instructions:

- Visit embed pages and verify that the layout is correct
- Try this with and without an `update` admin note present

### CODE REVIEW

File: client/stylesheets/shared/_embed.scss
Comment (joshuatf) [inline client/stylesheets/shared/_embed.scss:18]: One issue with applying this to the wrap is that store alerts will overlap with the header:

Screen Shot 2020-06-18 at 3 29 33 PM

**Review by joshuatf — COMMENTED** [review]
Thanks for working on this one, Bec!

Left a comment about overlapping I'm seeing.  I'm also thinking that the page titles may be a little too close to the header and there's a small "jump" that occurs after embed.

Screen Shot 2020-06-18 at 3 33 29 PM

Checking some of the Figma designs, I think we may need a little more space.

**Comment by becdetat** [comment]
> there's a small "jump" that occurs after embed

That jump occurs on main, I _think_ it has to do with something around [here](https://github.com/woocommerce/woocommerce-admin/blob/main/client/index.js#L48).

**Comment by becdetat** [comment]
Thanks @joshuatf , I've tweaked the CSS and it seems to work well with store alerts now

**Review by joshuatf — COMMENTED** [review]
Thanks for those changes, Bec!  That fixes the overlap for me on all pages.

The only issue I can see is possibly a rather large gap between page titles and the WCA header when store alerts don't exist or are disabled.  I'll let @jameskoster weigh in here and feel free to approve if things look good.

Screen Shot 2020-06-19 at 11 17 57 AM

**Comment by jameskoster** [comment]
Would it be possible for the gap between the screen title and the wc-admin header to match the gap between the screen title and the admin-bar on core screens?

Here's the post categories screen for reference:

categories

A `padding-top` value of 66px seems to bring us in to alignment:

**Comment by becdetat** [comment]
Ok the gap has been tightened in 80c954a, I'll put updated screenshots in the description.

**Review by joshuatf — COMMENTED** [review]
Code here looks good and margins look much better to me. @jameskoster would you mind giving this one last quick look when you have time?

**Review by jameskoster — APPROVED** [review]

**Comment by timmyc** [comment]
Rebased, going to close/open to get CI rolling again.

**Comment by timmyc** [comment]
hi

**Comment by timmyc** [comment]
Cherry picked to `release/1.3.0-rc.1` in e38281a

**Pull request merged**
